test/fixtures/poro.rb

```diff
@@ -258,4 +260,34 @@ def maker
   cache only: [:id]
   attributes :id
 end
+
+class TweetSerializer < ActiveModel::Serializer
+  attributes :id, :body, :date
+
+  belongs_to :author
+  class AuthorSerializer < ActiveModel::Serializer
+    attributes :id
+  end
```

Comment:
that's sorta how I see it as well. But for me, the biggest benefit would be to greatly reduce the number of top level serializers that actually have VERY specific purposes